tools/EMLassemblyline/templates1.xml

```diff
@@ -0,0 +1,70 @@
+<tool id="templates1" name="Make templates" version="0.1.0+galaxy0">
+    <description>for core features of a data package for EML metadata creation</description>
+    <requirements>
+        <requirement type="package" version="4.3.1">r-base</requirement>
+        <requirement type="package" version="3.5.5">r-emlassemblyline</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    
+        Rscript 
+         '$__tool_directory__/templates1.R'
+         '$license'
+         '$file_type'
+         
+    ]]></command>
+    <inputs>
+        <param name="license" type="select" label="Under what license are your data?">
+              <option value="CC0">CC0</option>
+              <option value="CCBY">CCBY</option>
+        </param>
+        <param name="file_type" type="select" label="What type of file do you want for templates?">
+              <option value=".txt">Text</option>
+              <option value=".docx">MS Word</option>
+              <option value=".md">Markdown</option>
+        </param>
+        
+    </inputs>
+    <outputs>
+          <collection name="output" type="list" label="Templates">
+              <discover_datasets pattern="__name_and_ext__"/>
+          </collection>
+    </outputs>
+    <tests>
+        <test>
+           <param name="license" value="CCBY"/>
+           <param name="file_type" value=".txt"/>
+           <output_collection name='output' type = "list" count="6"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+
+**What it does?**
+--------------------
+
+This tool produce templates for core features of a data package for EML metadata. 
+This tool can be used before of the make eml tool.
```

Comment:
Is there any advantage of having this as separate tool? Why not call this function as part of the MAKE EML tool?

Reply:
Because this is the classical work flow here. First creation of metadata templates, then you can modify it then you can write eml metadata file.